psst-gui/src/webapi/client.rs

```diff
@@ -285,6 +323,20 @@ impl WebApi {
         }
     }
 
+    pub fn get_track_credits(&self, track_id: &str) -> Result<TrackCredits, Error> {
+        let request = self
+            .get(
+                format!("track-credits-view/v0/experimental/{}/credits", track_id),
+                Some("spclient.wg.spotify.com"),
+            )?
+            .set(
+                "client-token",
+                self.session.client_token().unwrap_or_default(),
+            );
+        let response = Self::with_retry(|| Ok(request.clone().call()?))?;
```

Comment:
With_retry actually only applies when error 429 is sent back, which means too many requests have been sent. Is this needed?